doc/docs.md

```diff
@@ -8113,54 +8113,78 @@ instead use `#!/usr/bin/env -S v -raw-vsh-tmp-prefix tmp run`.
 
 ## Appendix I: Keywords
 
-V has 45 reserved keywords (3 are literals):
+V has 69 reserved keywords:
 
 ```v ignore
+!in
+!is
+__global
+__offsetof
 as
 asm
 assert
 atomic
+bool
 break
+byte
+char
 const
 continue
 defer
+dump
```

Comment:
dump is a bit special indeed 🤔, since it is implemented by the compiler, but behaves as a function from the point of view of the user